src/sagemaker/pytorch/README.rst

```diff
@@ -31,59 +31,65 @@ libraries installed. When creating the Estimator and calling deploy to create th
+| retrying                              | >=1.3.3       | >=1.3.3        | 1.3.3         | 1.3.3           | 1.3.3               |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| s3transfer                            | >=0.1.13      | >=0.1.13       | >=0.1.13      | 0.2.1           | 0.2.1               |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| sagemaker-containers                  | >=2.1.0       | >=2.1.0        | 2.4.10.post0  | 2.5.4           | 2.5.4               |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| sagemaker-inference (inference only)  | N/A           | N/A            | N/A           | N/A             | 1.1.2               |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| sagemaker-pytorch-container           | 1.0           | 1.1            | 1.2           | 1.2             | 1.3                 |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| setuptools                            | >=39.2.0      | >=40.4.3       | >=40.4.3      | 41.4.0          | 42.0.2.post20191203 |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| six                                   | >=1.11.0      | >=1.11.0       | 1.12.0        | 1.12.0          | 1.12.0              |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| torch                                 | 0.4.0         | 1.0.0          | 1.1.0         | 1.2.0           | 1.3.1               |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| torchvision                           | 0.2.1         | 0.2.1          | 0.3.0         | 0.4.0a0+9232c4a | 0.4.2               |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
+| Python                                | 2.7 or 3.5    | 2.7 or 3.6     | 2.7 or 3.6    | 2.7 or 3.6      | 2.7 or 3.6          |
++---------------------------------------+---------------+----------------+---------------+-----------------+---------------------+
 
 The Docker images extend Ubuntu 16.04.
 
-If you need to install other dependencies you can put them into `requirements.txt` file and put it in the source directory
+If you need to install other dependencies you can put them into ``requirements.txt`` file and put it in the source directory
```

Comment:
which doesn't work in all images anymore. Should we mention that maybe?

Reply:
good idea